packages/expo-image-picker/src/ExponentImagePicker.web.ts

```diff
@@ -75,7 +78,7 @@ function openFileBrowserAsync({
   mediaTypes,
   capture = false,
   allowsMultipleSelection = false,
-}: OpenFileBrowserOptions): Promise<ImagePickerResult> {
+}: OpenFileBrowserOptions): Promise<ImagePickerResult | ImagePickerMultipleResult> {
```

Comment:
I'd like to make this methods signature more explicit what's happening, just like you did in `ImagePicker.ts`.
```suggestion
function openFileBrowserAsync<T extends OpenFileBrowserOptions>({
  mediaTypes,
  capture = false,
  allowsMultipleSelection = false,
}: T): Promise<ExpandImagePickerResult<T>> {
```